parl/remote/job.py

```diff
@@ -239,6 +239,14 @@ def wait_for_files(self):
                 envdir = tempfile.mkdtemp()
                 for file in pyfiles:
                     code = pyfiles[file]
+
+                    # create directory (i.e. ./rom_files/)
```

Comment:
Why do we change job.py here? Is it for the support of Windows?

Reply:
For example, we want to send ``rom_files/pong.bin`` to the job. We need to create the ``rom_files`` folder at first. The old ``job.py`` only sends ``xxx.py`` files and doesn't create the folder.

Reply:
Could you add unit test for this part of code? what if the users pass ./pong.bin ? Does it still work?

Reply:
If we want to add a unit test, we need to change the  codes and get the ``tmp_dir`` at first. Then we can check if the ``./pong.bin`` is in the ``tmp_dir``.

Reply:
We can check by calling the function of the Actor, and check whether `./pong.bin` exists.

Reply:
```python
Class Actor(object):
	def check_local_file(self):
		return os.path.exist('./pong.bin')
      
```